engine/class_modules/priest/sc_priest.hpp

```diff
@@ -773,9 +775,10 @@ struct priest_action_t : public Base, public parse_buff_effects_t<priest_td_t>
     parse_buff_effects( p().buffs.dark_evangelism, p().talents.shadow.dark_evangelism );
     parse_buff_effects( p().buffs.surge_of_darkness, false );  // Mind Spike instant cast
     if ( p().is_ptr() )
-        parse_buff_effects( p().buffs.mind_melt, p().talents.shadow.mind_melt );                 // Mind Blast instant cast and Crit increase
+      parse_buff_effects( p().buffs.mind_melt,
+                          p().talents.shadow.mind_melt );  // Mind Blast instant cast and Crit increase
     else
-        parse_buff_effects( p().buffs.mind_melt );  // Mind Blast instant cast and Crit increase
+      parse_buff_effects( p().buffs.mind_melt );  // Mind Blast instant cast and Crit increase
```

Comment:
this entire indent change is very scary, could you just wrap it in brackets or just not touch it?

Reply:
wrapped in brackets